src/provisioning-ofono.c

```diff
@@ -32,6 +32,7 @@
 #define OFONO_MODEM_INTERFACE	OFONO_SERVICE ".Modem"
 #define OFONO_GPRS_INTERFACE	OFONO_SERVICE ".ConnectionManager"
 #define OFONO_CONTEXT_INTERFACE	OFONO_SERVICE ".ConnectionContext"
+#define MULTIMODEM
```

Comment:
I don't think there's a reason to keep this define. What works with more than one modem, will work with one modem too.